migration_scripts/bulk_load/src/hybrid_ev.clj

```diff
@@ -0,0 +1,587 @@
+           module                      TEXT,
+           tx_sequence_number          BIGINT,
+           event_sequence_number       BIGINT,
+           sender                      BYTEA
+       )")
+    (db/disable-autovacuum! tx +ev-emit-mod+)))
+
+(defn ev-emit-mod:constrain! [db]
+  (jdbc/with-transaction [tx db]
+    (db/with-table! tx +ev-emit-mod+
+      "ALTER TABLE %s
+       ADD PRIMARY KEY (
+           package,
+           module,
+           tx_sequence_number,
+           event_sequence_number
+       ),
+       ALTER COLUMN sender SET NOT NULL")))
+
+(defn ev-emit-mod:index! [db]
+  (db/with-table! db +ev-emit-mod+
+    "CREATE INDEX %1$s_sender ON %s (
+         sender,
+         package,
+         module,
+         tx_sequence_number,
+         event_sequence_number
+     )"))
+
+;; Table: hybrid_ev_event_pkg ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defn ev-event-pkg:create! [db]
+  (jdbc/with-transaction [tx db]
+    (db/with-table! db +ev-event-pkg+
+      "CREATE TABLE %s (
+           package                     BYTEA,
+           tx_sequence_number          BIGINT,
+           event_sequence_number       BIGINT,
+           sender                      BYTEA
+       )")
+    (db/disable-autovacuum! tx +ev-event-pkg+)))
+
+(defn ev-event-pkg:constrain! [db]
+  (jdbc/with-transaction [tx db]
+    (db/with-table! tx +ev-event-pkg+
+      "ALTER TABLE %s
+       ADD PRIMARY KEY (
+           package,
+           tx_sequence_number,
+           event_sequence_number
+       ),
+       ALTER COLUMN sender SET NOT NULL")))
+
+(defn ev-event-pkg:index! [db]
+  (db/with-table! db +ev-event-pkg+
+    "CREATE INDEX %1$s_sender ON %s (
+         sender,
+         package,
+         tx_sequence_number,
+         event_sequence_number
+     )"))
+
+;; Table: hybrid_ev_event_mod ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defn ev-event-mod:create! [db]
+  (jdbc/with-transaction [tx db]
+    (db/with-table! db +ev-event-mod+
+      "CREATE TABLE %s (
+           package                     BYTEA,
+           module                      TEXT,
+           tx_sequence_number          BIGINT,
+           event_sequence_number       BIGINT,
+           sender                      BYTEA
+       )")
+    (db/disable-autovacuum! tx +ev-event-mod+)))
+
+(defn ev-event-mod:constrain! [db]
+  (jdbc/with-transaction [tx db]
+    (db/with-table! tx +ev-event-mod+
+      "ALTER TABLE %s
+       ADD PRIMARY KEY (
+           package,
+           module,
+           tx_sequence_number,
+           event_sequence_number
+       ),
+       ALTER COLUMN sender SET NOT NULL")))
+
+(defn ev-event-mod:index! [db]
+  (db/with-table! db +ev-event-mod+
+    "CREATE INDEX %1$s_sender ON %s (
+         sender,
+         package,
+         module,
+         tx_sequence_number,
+         event_sequence_number
+     )"))
+
+;; Table: hybrid_ev_event_name ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
In this table, `name` is just the type name, without type parameters, e.g. `CoinMetadata`.